build/v1/types.go

```diff
@@ -1016,6 +1022,29 @@ type ImageChangeTrigger struct {
 	Paused bool `json:"paused,omitempty" protobuf:"varint,3,opt,name=paused"`
 }
 
+// ImageChangeTriggerStatus tracks the latest resolved status of the associated ImageChangeTrigger policy
+// specified in the BuildConfigSpec.Triggers struct.
+type ImageChangeTriggerStatus struct {
+	// lastTriggeredImageID contains the resolved DockerImage sha from the ImageStreamTag referenced in the
+	// ImageChangeTrigger.From field when that trigger resulted in a Build being initiated.
+	LastTriggeredImageID string `json:"lastTriggeredImageID,omitempty" protobuf:"bytes,1,opt,name=lastTriggeredImageID"`
+
+	// from is whichever ImageStreamTag, either an explicitly cited ImageChangeTrigger ImageStreamTag in the spec,
+	// or the From setting in the build strategy if the From field in the ImageChangeTrigger is not set, that resulted in
+	// the triggering of a build.  This field can be used to correlate with specific ImageChangeTrigger policicy specifications
```

Comment:
```suggestion
	// the triggering of a build.  This field can be used to correlate with specific ImageChangeTrigger policy specifications
```

Reply:
looks like I have to pull this in and run make update to make the verify job happy